lymph/unilateral.py

```diff
@@ -788,6 +787,34 @@ def _evolve(
         state_probs[-1] = state
 
         return state_probs
+    
+    def _evolve_onestep(

```

Comment:
I think, as with the `_evolve_midext()` function, this computes the entire evolution, right? But this is already implemented, so you can just use that. Or is there something missing from the original implementation?